daprdocs/content/en/reference/cli/dapr-run.md

```diff
@@ -40,7 +40,7 @@ dapr run [flags] [command]
 | `--profile-port`               |                      | `7777`                                                                             | The port for the profile server to listen on                                                         |
 | `--unix-domain-socket`, `-u`   |                      |                                                                                    |  Path to a unix domain socket dir mount. If specified, communication with the Dapr sidecar uses unix domain sockets for lower latency and greater throughput when compared to using TCP ports. Not available on Windows OS |
 | `--dapr-http-max-request-size` |                      | `4`                                                                                | Max size of request body in MB.                                                                      |
-
+| `--dapr-http-read-buffer-size` |                      | `4`                                                                                | max size of read buffer in KB.                                                                      |
```

Comment:
Max size of

Reply:
Thank you very much, I need to use this configuration. Looking at the original document, I found that there is no relevant introduction. By viewing the relevant source code, I found that there is a configuration, so I supplemented the document by the way.I would like to be corrected if there are any mistakes.

Reply:
@Taction @apuzyrevsky